java/light-4j/config.yaml

```diff
@@ -0,0 +1,14 @@
+framework:
+  website: doc.networknt.com
+  version: 1.6.23
+
+build:
+  - mvn clean package -Prelease
```

Comment:
what does `-Prelease` mean ?

Reply:
-Prelease is a profile in the generated pom.xml to build a fat jar. During the development cycle, we don't build fat jar at all to save time for developers. The fat jar will be built only before release.